packages/quantic/force-app/examples/main/lwc/exampleInsightPanel/resultTemplates/youtubeResultTemplate.html

```diff
@@ -1,6 +1,6 @@
 <template>
-  <c-quantic-result-template is-any-preview-open={isAnyPreviewOpen} result-preview-should-not-be-accessible={resultPreviewShouldNotBeAccessible}>
-    <c-quantic-result-label slot="label" result={result}></c-quantic-result-label>
+  <c-quantic-result-template is-any-preview-open={isAnyPreviewOpen} result-preview-should-not-be-accessible={resultPreviewShouldNotBeAccessible} hide-label>
+    <c-quantic-colored-result-badge label={result.raw.objecttype}></c-quantic-colored-result-badge>
```

Comment:
This component is not assigned to any slot of the Quantic Result Template component.it needs to be assigned to a slot, otherwise it won't appear.